model/session.go

```diff
@@ -29,6 +29,7 @@ type Session struct {
 	LastActivityAt int64     `json:"last_activity_at"`
 	UserId         string    `json:"user_id"`
 	TeamId         string    `json:"team_id"`
+	TeamURL        string    `json:"team_url"`
```

Comment:
Yeah and I think changing the TeamUrl in the Context struct to be just TeamName or TeamId would be useful as well because there will be requests that don't have a session but are still doing something on a specific team